incubator/cloudflare-datadog/templates/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "fullname" . }}
```

Comment:
Please apply best practices for lablels.
https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices
The `app` label should be `{{ template "name" . }}`.